### PR TITLE
[Mellanox] Revise label name and fix typo in sensor.conf of 4600C

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
#### Why I did it

Revise lable name and fix typo in sensor.conf of 4600C

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

Revise lable name and fix typo in sensor.conf of 4600C

#### How to verify it

Manual test
sonic-mgmt test_sensors.py

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [x] 202205
- [x] 202211
- [x] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Review by keboliu — APPROVED** [review]

**Review by liat-grozovik — DISMISSED** [review]

File: device/mellanox/x86_64-mlnx_msn4600c-r0/sensors_respin.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ##
-## Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES.
+## Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
```
[the remark this answers is not in the document]
Comment (Junchao-Mellanox) [reply device/mellanox/x86_64-mlnx_msn4600c-r0/sensors_respin.conf]: fixed

**Review by liat-grozovik — APPROVED** [review]

**Comment by Junchao-Mellanox** [comment]
Hi @StormLiangMS , could you please help cherry-pick to 202211?
Hi @yxieca , could you please help cherry-pick to 202205?

**Comment by bingwang-ms** [comment]
@Junchao-Mellanox Is this PR required for 202205 branch? We didn't notice any test failure.

**Comment by liat-grozovik** [comment]
> @Junchao-Mellanox Is this PR required for 202205 branch? We didn't notice any test failure.

yes it is.

**Comment by Junchao-Mellanox** [comment]
Hi @bingwang-ms , yes. It fixed some typo and duplicate label name. You will also need sonic-mgmt PR: https://github.com/sonic-net/sonic-mgmt/pull/9824 which fixed the sensor configuration file.

**Comment by Junchao-Mellanox** [comment]
Hi @StormLiangMS , can you please help cherry-pick to 202211?

**Pull request merged**